docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -77,7 +77,7 @@ Some key points:
 * Keptn provides an operator
   that can observe and orchestrate application-aware workload life cycles.
   This operator leverages Kubernetes webhooks
-  and extends the Kubernetes scheduler
+  and the Kubernetes scheduler
```

Comment:
```suggestion
  and the Kubernetes scheduler (if installed)
```

Reply:
in both cases we leverage the Kubernetes scheduler , this is always installed by k8s